engine/engine.go

```diff
@@ -6,8 +6,11 @@ package engine
 
 import (
 	"context"
+	goerrors "errors"
```

Comment:
Is this renaming still needed?

Reply:
`errors` is used for an internal package a few lines below